### ISSUE TITLE
Support for Temperature and Humidity Sensor...

### ISSUE THREAD

**Issue by ctschach** [jchassin/homebridge-owfs#3]
I have a combined sensor which delivers both, temperature and humidity. It would be great if home bridge-owfs could identify this and also add the humidity as a device. 

Here is my output of powder:

root@knx:/home# owdir /uncached/26.935D27010000
/uncached/26.935D27010000/B1-R1-A
/uncached/26.935D27010000/CA
/uncached/26.935D27010000/EE
/uncached/26.935D27010000/HIH3600
/uncached/26.935D27010000/HIH4000
/uncached/26.935D27010000/HTM1735
/uncached/26.935D27010000/IAD
/uncached/26.935D27010000/MultiSensor
/uncached/26.935D27010000/S3-R1-A
/uncached/26.935D27010000/VAD
/uncached/26.935D27010000/VDD
/uncached/26.935D27010000/address
/uncached/26.935D27010000/alias
/uncached/26.935D27010000/crc8
/uncached/26.935D27010000/date
/uncached/26.935D27010000/disconnect
/uncached/26.935D27010000/endcharge
/uncached/26.935D27010000/family
/uncached/26.935D27010000/humidity
/uncached/26.935D27010000/id
/uncached/26.935D27010000/locator
/uncached/26.935D27010000/offset
/uncached/26.935D27010000/pages
/uncached/26.935D27010000/r_address
/uncached/26.935D27010000/r_id
/uncached/26.935D27010000/r_locator
/uncached/26.935D27010000/temperature
/uncached/26.935D27010000/type
/uncached/26.935D27010000/udate
/uncached/26.935D27010000/vis

**Comment by jchassin** [comment]
HI could you tell me a little about the hardware you re using ?

**Comment by rnilssoncx** [comment]
I've added support for the following:
OWFS_DS2438 - Humidity/Temp
OWFS_EDS0065 - Humidity/Temp

What's the best way to get you the code?

Rob.

**Comment by jchassin** [comment]
Hi Robert, the best is to fork, add tou code and to send a pull request
Jean

Le jeu. 17 mai 2018 à 20:17, Robert Nilsson <notifications@github.com> a
écrit :

**Comment by rnilssoncx** [comment]
Done!

**Comment by jchassin** [comment]
Thanks you, would you share the schematics of these devices ?
I have limited access to my board, I'll perform the merge and publish it next week
Jean

**Comment by jchassin** [comment]
Can you update the README file with some configuration samples ? (as per done for 1820....)
Tks,
Jean

**Comment by rnilssoncx** [comment]
Jean,

I made some major changes to the plugin.  Rather than lock in the sensors supported, I made that configurable.  As long as the sensor has temperature or humidity in a field, it should work.

I also added support for fakegago-history (https://github.com/simont77/fakegato-history <https://github.com/simont77/fakegato-history>) - this provides nice graphing in the Elgato Eve app.  Not sure if any other apps are using this yet.

Here’s the link to the ESD0065:  http://owfs.org/index.php?page=eds0065---temperature-humidity <http://owfs.org/index.php?page=eds0065---temperature-humidity>
It does a lot more than just read temperature - it’s a control as well.  I use them to run my dehumidifiers.

With the altered structure of the plugin it should be easy to add light sensors, for example.

I’ve tested this in my environment and it’s stable.  Let me know if you have any questions.

Thanks!

Rob.
